include/libzfs.h

```diff
@@ -408,6 +408,7 @@ typedef struct importargs {
 	int unique : 1;		/* does 'poolname' already exist?	*/
 	int exists : 1;		/* set on return if pool already exists	*/
 	int scan : 1;		/* prefer scanning to libblkid cache    */
+	int do_path : 1;	/* use specified device paths		*/
```

Comment:
nit: `path` would be more consistent with the other variable names for this, although I think `fullpaths` would be better.